services/src/main/java/io/grpc/services/BinaryLog.java

```diff
@@ -244,4 +265,101 @@ public BinaryLog getLog(String fullMethodName) {
       return null;
     }
   }
+
+  /**
+   * Returns a {@link Uint128} by interpreting the first 8 bytes as the high int64 and the second
+   * 8 bytes as the low int64.
+   */
+  // TODO(zpencer): verify int64 representation with other gRPC languages
+  static Uint128 callIdToProto(byte[] bytes) {
+    Preconditions.checkArgument(bytes.length == 16);
```

Comment:
include the actual length of the number of bytes in the error message.

Reply:
done